File: GymAppV2.Core/Entities/Trainer.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GymAppV2.Core.Entities
{
    public class Trainer : BaseEntity
    {
        public string Firstname { get; set; } = string.Empty;
        public string Lastname { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public string Specialization { get; set; } = string.Empty;
        public bool IsActive { get; set; } = true;
        public ICollection<GymProgram> Programs { get; set; } = new List<GymProgram>();
    }
}

File: gymapp-client/src/app/core/models/trainer.ts
export interface Trainer {
  id: number;
  firstname: string;
  lastname: string;
  email: string;
  specialization: string;
  isActive: boolean;
}

export interface CreateTrainer {
  firstname: string;
  lastname: string;
  email: string;
  specialization: string;
}

export interface UpdateTrainer {
  firstname: string;
  lastname: string;
  specialization: string;
}
